pyCRISPRcleanR/crisprCleanR_command.py

```diff
@@ -52,8 +52,8 @@ def main():  # pragma: no cover
     optional.add_argument("-np", "--num_processors", type=int, dest="num_processors", required=False,
                           default=1, help="Number of processors to use for parallel jobs")
 
-    optional.add_argument("-cc", "--correct_counts", type=int, dest="correct_counts", required=False,
-                          default=None, help="Correct counts using CBS [Y/y]")
+    optional.add_argument("-cc", "--crispr_cleanr", type=str, dest="crispr_cleanr", required=False,
```

Comment:
Could this have been a flag rather than having a value? `action='store_true'`. Non essential, just a query

Reply:
made this change